roles/registrator/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+- name: wait for weave socket to be ready.
+  wait_for:
+    port: 6783
+    delay: 10
+
+- name: destroy old marathon container
```

Comment:
if we have a systemd init file then we dont need either of these tasks - just deploy the init file and start it. no need for rebuild_container stuff